### PR TITLE
ASNIDE-135 Added real ASN.1 snippets instead of stubs

### PR DESCRIPTION
**Pull request by psockoN7**
The important thing is that I have change completion to start after typing two signs instead of three. I have done it,  to support _set_ snippet. Snippets proposals hide, after typing full snippet trigger, so when snippets proposals were trigerred with with three letters, than it was not possible to use _set_ snippet. The user experience comming from this change should be monitored.

### CODE REVIEW

File: snippets/asn.xml [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
-        trackingNumber INTEGER,
-        question       IA5String
-    }
-
-    Test ::= SEQUENCE {
-        other $name$
-    }
-
-END
+<snippet group="ASN.1" trigger="sequence" id="sequence">SEQUENCE
```
Comment (hcorg) [inline snippets/asn.xml]: I'm not sure, but I had some errors while using the same  id across different snippets files/groups. So ID might need to be globally unique. Needs verification. If so, some prefix like `Asn1Acn.Asn1.` could be added

File: snippets/asn.xml [hunk 2]
```diff
@@ -1,36 +1,30 @@
...
+{
+    $$
+}
+</snippet>
+<snippet group="ASN.1" trigger="enumerated" id="enumerated">ENUMERATED
```
Comment (hcorg) [inline snippets/asn.xml]: I wonder - should we create snippets for what is after `::=` or to include assignment in snippet. Or maybe we should come out with some convention and add snippets for "enumerated + assignment", something like "asequence"

File: snippets/asn.xml [hunk 3]
```diff
@@ -1,36 +1,30 @@
...
-        answer         BOOLEAN
-    }
-
-    History ::= SEQUENCE {
-        questions SEQUENCE(SIZE(0..10)) OF Question,
-        answers   SEQUENCE(SIZE(1..10)) OF Answer,
-        anArray   SEQUENCE(SIZE(100))  OF INTEGER(0..1000),
-    }
-
+<snippet group="ASN.1" trigger="setof" id="setof">SET (SIZE($constraint1$ .. $constraint2$)) OF $type$</snippet>
```
Comment (hcorg) [inline snippets/asn.xml]: I'd go with single `constaint` - it can be `(1 .. 2)` or `(1, 2, 3)` or `(3)`, let the user decide

Comment (psockoN7) [reply snippets/asn.xml]: ok.

File: snippets/asn.xml (hunk 1 above)
Comment (psockoN7) [reply snippets/asn.xml]: This is true, I have tested and duplicated ids might not be working. I will follow prefixing rule shared among other snippet files, which is using underscores.

File: snippets/asn.xml (hunk 2 above)
Comment (psockoN7) [reply snippets/asn.xml]: added 'with assignment' versions of snippets

**Pull request merged**